helm/values.yaml

```diff
@@ -83,6 +83,26 @@ minio:
           target:
             averageValue: 150m
             type: AverageValue
+
+  externalscaler:
+    enabled: false
+    image: docker.pkg.github.com/cwm-keda-external-scaler/iamazeem/cwm-keda-external-scaler
```

Comment:
the external scaler image should be automatically updated by CI on every push to cwm-keda-external-scaler main branch

Reply:
With the latest tag or with Github SHA?
It should act as a trigger to invoke the CI, right?

Reply:
with github sha, same as logger image

Reply:
Added with https://github.com/iamazeem/cwm-keda-external-scaler/pull/1

Reply:
see comment on that PR, it should work exactly the same as the logger image update